galactic/distribution.yaml

```diff
@@ -1523,6 +1523,12 @@ repositories:
       url: https://github.com/ros2/message_filters.git
       version: galactic
     status: maintained
+  micro_ros_msgs:
+    release:
```

Comment:
could you add `source` and `doc` entries?
it's not a requirement, but it's usually done

Reply:
Done